apps/frontend/core-dapp/src/features/simulator/SimulatorCard.tsx

```diff
@@ -52,6 +55,15 @@ const MarketNotAvailable = styled.span`
   color: ${({ theme }): string => theme.color.neutral2};
 `
 
+const SimulatorSkeleton = styled(Skeleton).attrs({
+  width: '232px',
+  height: '136px',
+})`
```

Comment:
skeleton has a weird width on mobile, could we try `width: '100%'`?

```suggestion
const SimulatorSkeleton = styled(Skeleton).attrs({
  width: '100%',
  height: '136px',
})`
```

Reply:
Funny, I tried this initially but it didn't work for some reason. It does work now.